addons/payment/tests/test_payment_provider.py

```diff
@@ -1,6 +1,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo.tests import tagged
+from unittest.mock import patch
```

Comment:
Import of standard libraries before those from Odoo (sorry, didn't saw it before '^^)